packages/roosterjs-editor-plugins/lib/plugins/Announce/AnnounceHandlerImpl.ts

```diff
@@ -0,0 +1,69 @@
+import type { AnnounceData, DefaultAnnounceStrings } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ */
+export default class AnnounceHandler {
+    private ariaLiveElement: HTMLDivElement | undefined;
+
+    constructor(
+        private readonly document: Document,
+        private stringsMap?: Map<DefaultAnnounceStrings, string>
+    ) {}
+
+    public announce(announceData: AnnounceData) {
+        const { text, defaultStrings, formatStrings = [] } = announceData;
+        let textToAnnounce = formatString(
+            (defaultStrings && this.stringsMap?.get(defaultStrings)) || text,
```

Comment:
What if a 3rd part feature also want to use announce, do they need to modify DefaultAnnounceStrings enum?

Maybe just use a string or string-enum instead, so that 3rd party code can also use its own string key, similar with `ChangeSource`

Reply:
In that case they can use the "text" string property from the AnnounceData instead. 
I will update the comment with this info.